.github/workflows/release.yaml

```diff
@@ -0,0 +1,22 @@
+name: Release
+on: workflow_dispatch
+
+jobs:
+  windows:
+    name: Windows Release
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: |
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+        name: git config
+      - run: npm ci
+      # See .release-it.yml for build steps that it will run after bumping the version
+      - run: npx release-it -VV --ci --github.release=true
```

Comment:
If we're going to use `npx` here, any reason to have a devDep on the package in the package.json?

Reply:
I thought `npx` merely runs an *existing* binary from node_modules/.bin... it won't go out and install it and then run it... so something still needs to cause it to be installed. If you'd prefer, I could add a task in package.json instead of calling `npx` directly. I chose not to to reduce the chance someone would accidentally call it.

Reply:
It will run that one if it exists, if not it will install it so this works to make sure it uses a more consistent version.